mlir/lib/Tools/PDLL/Parser/Parser.cpp

```diff
@@ -3048,6 +3166,38 @@ Parser::createTupleExpr(SMRange loc, ArrayRef<ast::Expr *> elements,
   return ast::TupleExpr::create(ctx, loc, elements, elementNames);
 }
 
+FailureOr<ast::DeclRefExpr *>
+Parser::createNativeCall(SMRange loc, StringRef nativeFuncName,
+                         MutableArrayRef<ast::Expr *> arguments) {
+
+  FailureOr<ast::Expr *> nativeFuncExpr = parseDeclRefExpr(nativeFuncName, loc);
+  if (failed(nativeFuncExpr))
+    return emitError(nativeFuncName + " not found.");
+
+  if (!(*nativeFuncExpr)->getType().isa<ast::RewriteType>())
+    return emitError(nativeFuncName + " should be defined as a rewriter.");
```

Comment:
According to https://mlir.llvm.org/deprecation/ we should use the free function variant `isa<T>()` for this if possible

Reply:
I agree, but it doesn't seem that can be done for this case.